.github/workflows/migration-tests.yml

```diff
@@ -87,7 +75,7 @@ jobs:
   
   config:
     needs: [ change-detection ]
```

Comment:
maybe set-output had to be included in needs in the jobs using it ?

Reply:
tried that as well, but the conditions are not exactly same in each step, for example it is different in change-detection vs checkout. I've now kept it simple but we still get to re-use `shouldTriggerResourceTest` in all tests
Hopefully this works in this repo too :v: